sdk/monitor/Azure.ResourceManager.Monitor/src/Generated/ArmMonitorModelFactory.cs

```diff
@@ -1726,5 +1726,29 @@ public static MonitorWorkspaceResourceData MonitorWorkspaceResourceData(Resource
         {
             return MonitorWorkspaceResourceData(id: id, name: name, resourceType: resourceType, systemData: systemData, tags: tags, location: location, etag: etag, defaultIngestionSettings: defaultIngestionSettings, metrics: metrics, accountId: accountId, metricsPropertiesMetrics: default, provisioningState: provisioningState, defaultIngestionSettingsPropertiesDefaultIngestionSettings: default, privateEndpointConnections: privateEndpointConnections, publicNetworkAccess: publicNetworkAccess);
         }
+
```

Comment:
actually I changes my mind. For this package, just supress these violations using [assembly: SuppressMessage(...)]